### PR TITLE
LEST-NONE - Remove minification

### PR DESCRIPTION
**Pull request by yogwoggf**
## Changes

Removes the minify process from the lest package build.

## Impact

Makes it larger but makes debugging MUCH easier.

## Testing

- Build lest
- Review the output `lest.lua`
- See that each bundled lua file is an exact copy of the corresponding source file

## Helpful Links

[API Docs](https://taservers.github.io/lest/docs/api/expect)

[Project Board](https://taservers.atlassian.net/jira/software/c/projects/LEST/boards/4)

[Discord](https://discord.tasevers.com)

### CODE REVIEW

File: packages/lest/package.json [hunk 1]
```diff
@@ -19,8 +19,7 @@
   },
   "scripts": {
     "bundle": "luabundler bundle src/lua/lest.lua -p \"./?.lua\" -o dist/lua/lest.lua",
-    "minify": "lua scripts/minify.lua",
-    "build:lua": "npm run bundle && npm run minify",
+    "build:lua": "npm run bundle",
```
Comment (Derpius) [inline packages/lest/package.json]: Does the CI call the `bundle` script? If not can we merge them into just `build:lua`?

Comment (yogwoggf) [reply packages/lest/package.json]: Nope, CI is completely oblivious to all of our details related to actual packaging, it just calls `npm run build:lest` from the root dir

**Review by Derpius — APPROVED** [review]

**Comment by Derpius** [comment]
One thing you need to do actually, is rebase with commit messages that will trigger a release. Can squash these into “fix: remove minifying to make stack traces readable”

**Comment by yogwoggf** [comment]
Ok.

**Review by Derpius — APPROVED** [review]

**Pull request merged**